lib/actions/ui.js

```diff
@@ -230,3 +234,37 @@ export const MobileScreens = {
   SET_DATETIME: 7,
   RESULTS_SUMMARY: 8
 }
+
+/**
+ * Enum to describe the layout of the itinerary view
+ * (currently only used in batch results).
+ */
+export const ItineraryView = {
+  DEFAULT: 'list',
+  FULL: 'full',
+  HIDDEN: 'hidden',
+  LEG: 'leg',
+  LIST: 'list'
```

Comment:
Can each of these have a comment about what they mean?

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/339/commits/cf678bbf51da21206f401c18e87cb719c805f6ff.